pyang/statements.py

```diff
@@ -330,7 +329,13 @@ class Abort(Exception):
     'mandatory':True,
     'min-elements':True,
     'max-elements':True
-    }
+}
+
+_deviate_delete_singleton_keywords = {
+    'type':True,
```

Comment:
It is not legal to delete 'type'

Reply:
Yeah, you are right. I will modify this and add some testcases tomorrow.